lib/sort_by_length.rb

```diff
@@ -1,7 +1,35 @@
 # A method which will return an array of the words in the string
 #  sorted by the length of the word.
-# Time complexity: ?
+# Time complexity: 0(n) n being the length - a loop based off of n
 # Space complexity: ?
+
 def sort_by_length(my_sentence)
```

Comment:
Nice implementation of the smart bubblesort.  Also the space complexity is O(n) because you split the sentence.